lib/src/main/java/com/squareup/cycler/RecyclerMutations.kt

```diff
@@ -14,11 +19,14 @@ fun <I : Any> Recycler.Config<I>.enableMutations(block: MutationExtensionSpec<I>
 class MutationExtensionSpec<T : Any> : ExtensionSpec<T> {
   override fun create(): Extension<T> = MutationExtension(this)
   internal var onMove: ((Int, Int) -> Unit)? = null
-  internal var onSwipeToRemove: ((T) -> Unit)? = null
+  internal var onSwipeToRemove: ((T, direction: Int) -> Unit)? = null
```

Comment:
Should this be renamed `onSwipe`? `onSwipeToRemove` feels a little prescriptive for what the swipe should do?

Reply:
Yes. I don't remember when we introduced that. I don't know if the removal is automatic or not (I think it is). But even if it is, it could be called just swipe.

I'll look into that in a separate branch in the near future.